gremlin-core/pom.xml

```diff
@@ -37,7 +41,7 @@ limitations under the License.
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
-            <version>1.15</version>
+            <version>${snakeyaml.version}</version>
```

Comment:
any reason you turned these into properties? doesn't seem necessary as they aren't really used anywhere outside of gremlin-core.  i think we tend to only use properties in the root pom.xml when there's a need to keep a value consistent that might be referenced in multiple places.